form/form.go

```diff
@@ -18,6 +18,19 @@ import (
 	"gopkg.in/juju/environschema.v1"
 )
 
+// Form describes a form based on a schema.
+type Form struct {
+	Title  string
```

Comment:
// Title holds the title of the form, giving contextual information
// for the fields.

?